include/lmct_config.h

```diff
@@ -29,7 +34,11 @@ struct lmct_config {
 
 extern struct lmct_config lmct_conf;
 
+extern enum filter_type fltr_type;
 void
 init_lmct_config();
 
+void
```

Comment:
on another note, let us move this to common.h and the corresponding implementation in common.c instead.

`lmct_config` is basically something which is read from the conf file and populated.
`filter_type` is read from args instead.